src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -782,7 +785,10 @@ public static void Main()
             comp.VerifyDiagnostics(
                 // (8,22): error CS1525: Invalid expression term '='
                 //         c = c with { = 5 };
-                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "=").WithArguments("=").WithLocation(8, 22)
+                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "=").WithArguments("=").WithLocation(8, 22),
+                // (8,22): error CS1061: 'C' does not contain a definition for '' and no accessible extension method '' accepting a first argument of type 'C' could be found (are you missing a using directive or an assembly reference?)
+                //         c = c with { = 5 };
+                Diagnostic(ErrorCode.ERR_NoSuchMemberOrExtension, "").WithArguments("C", "").WithLocation(8, 22)
```

Comment:
>ERR_NoSuchMemberOrExtension [](start = 37, length = 27)

This feels like a cascaded diagnostic. Please fix it or take a note of that for possible future improvement.

Reply:
Yup, I have a follow-up PR that should address it.